demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1968,29 +1969,91 @@ def validate_job(self, structure_validator, pack_error_ignore_list):
 
         return is_valid
 
+    def get_old_file_path(self, file_path):
+        """
+        Extract the old file path from the given file
+        Args:
+            file_path: the path to extract the old file path from.
+        """
+        return file_path[:2] if isinstance(file_path, tuple) else (file_path, file_path)
+
+    def get_all_files_edited_in_pack_ignore(self, modified_files: set) -> set:
+        """
+        Extract all the files the file paths of files that there pack-ignore section was ignored somehow.
+        Args:
+            modified_files: The list of modified files.
+        """
+        all_files = self.git_util.get_all_files()
+        all_files_edited_in_pack_ignore: set = set()
+        for file_path in modified_files:
+            # handle renamed files
+            old_file_path, file_path = self.get_old_file_path(file_path)
+            if not file_path.endswith(".pack-ignore"):
+                continue
+            old_file_content = get_remote_file(old_file_path, tag="master")
+            config = ConfigParser(allow_no_value=True)
+            config.read_string(old_file_content)
+            old_file_content = self.get_error_ignore_list(config=config)
+            pack_name = get_pack_name(str(file_path))
+            file_content = self.get_error_ignore_list(pack_name)
+            files_to_test = set()
+            for key, value in old_file_content.items():
+                if not (section_values := file_content.get(key, [])) or not set(
+                    section_values
+                ) == set(value):
+                    files_to_test.add(key)
+            for key, value in file_content.items():
+                if not (section_values := old_file_content.get(key, [])) or not set(
+                    section_values
+                ) == set(value):
+                    files_to_test.add(key)
+
+            def is_relevant_file(file: Path, pack_name: str) -> bool:
+                """
+                Return wether the given file is under the given pack
+                Args:
+                    file: The file to check.
+                    pack_name: The name of the pack we want to ensure the given file is under.
+                """
+                return (
```

Comment:
do we not have another function that checks if a file is in a pack/ it doesnt come into this function only under a certain pack?

Reply:
I tried to search tools.py but couldn't find anything..